### PR TITLE
Ticket 3

### PR DESCRIPTION
**Pull request by Sarahxbkelly**
Added new feature to address ticket 3:

Ignore empty lines

Description Update your input script to ignore any empty lines found when reading in the
input data file.

Objectives 

● A final array is created with any empty lines omitted.

### CODE REVIEW

**Review by SuperMattyG — COMMENTED** [review]
Hi,  what does the all in the if statement  'if all(rec):' actually do?

**Review by SuperMattyG — COMMENTED** [review]
This looks good to me, however if the input csv file were to massively grow in the future then the .append would start to take a while to process it.

**Review by SuperMattyG — COMMENTED** [review]
Some nice test cases for the first 3 tickets

**Review by SuperMattyG — COMMENTED** [review]
Go Ahead

**Comment by Sarahxbkelly** [comment]
@SuperMattyG the all is a boolean function and for any empty iterables like lists, tuples, and strings it will provide a truth value of False. For any non-empty values it provides True. I've just created a unit test for this too so will add it for review. 

Good shout on the .append - do you have any ideas to make this more efficient?

File: python-survey-app/remove_empty_record.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def remove_empty_records(dataset):
+
+    results = []
+
+    for rec in dataset:
+        if all(rec):
+            results.append(rec)
+
+    return results
```
Comment (SuperMattyG) [inline python-survey-app/remove_empty_record.py:9]: There is no new line at the end of this def.

Should be there fore readability purposes.

**Comment by SuperMattyG** [comment]
> This looks good to me, however if the input csv file were to massively grow in the future then the .append would start to take a while to process it.

Consider using extend() method https://stackoverflow.com/questions/65573288/how-to-build-a-list-faster-than-append

**Pull request merged**